test/recipes/15-test_rsa.t

```diff
@@ -49,7 +49,7 @@ sub run_rsa_tests {
 
      SKIP: {
          skip "Skipping msblob conversion test", 1
-	     if disabled($cmd) || disabled("dsa") || $cmd == 'pkey';
+	     if disabled($cmd) || disabled("dsa") || $cmd eq 'pkey';
```

Comment:
I am confused by the `disabled("dsa")` here. Why is this supposed to not work with disabled DSA when the key is RSA.

Reply:
Ah, I suppose that's because before #13648 the MS encoders were disabled by OPENSSL_NO_DSA. So perhaps this should be fixed up here as well?

Reply:
Yes!  Thanks for the reminder